openwisp_notifications/templates/openwisp_notifications/email_template.html

```diff
@@ -0,0 +1,95 @@
+{% load i18n %}
+{% load l10n %}
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  {% block styles %}
+  <style type="text/css">
+    .background {
+      height: 100%;
+      background: linear-gradient(to bottom, #bb442c 50%, #464646 50%);
+      background-repeat: no-repeat;
+      background-attachment: fixed;
+      padding: 50px;
+    }
+
+    .container {
+      padding: 20px;
+      width: 75%;
+      margin: auto;
+      background-color: whitesmoke;
+      border-radius: 10px;
+    }
+
+    .notification-description {
+      padding: 0px 30px;
+    }
+
+    .logo {
+      width: 100px;
+    }
+
+    table {
+      border-collapse: collapse;
+    }
+
+    td,
+    th {
+      padding: 10px;
+    }
+
+    .notification-header {
+      background-color: #464646;
+      color: white;
+    }
+
+    .notification-subject {
+      width: 100%;
+      text-align: right;
+    }
+
+    .notification-subject a {
+      color: white;
+    }
+  </style>
+  {% endblock styles %}
+</head>
+
+<body>
+  {% block body %}
+  <div class="background">
+    <div class="container">
+      <table style="width: 100%;">
+        <thead class="notification-header">
+        {% block notification_header %}
+          <tr>
+            <th><img src="{{ OPENWISP_NOTIFICATION_EMAIL_LOGO }}" alt="Logo" id="logo" class="logo"></th>
+            <th class="notification-subject">{{ notification.email_subject }}</th>
+          </tr>
+        {% endblock notification_header %}
+        </thead>
+        <tbody>
+          {% block notification_body %}
+          <tr class="notification-body">
+            <td colspan="2">
+              <p> {% trans 'Greetings' %} {{ notification.recipient.username }},</p>
+              <p> {% trans 'You have a new notification from your network:' %} {{ site }}</p>
+              <div class="notification-description">
+                <p> {{ notification.message }} </p>
+              </div>
+              <p>{% trans 'You can also view this notifification on the dashboard by following' %}
+                <a href="{{ site.domain }}{{ admin_url }}" id="admin-url">{% trans ' this link' %}.</a>
```

Comment:
don't use "this link" and don't leave extra space.

I suggest building the URL outside of the template (like it's done for the plain text).